prac_05/hex_colours.py

```diff
@@ -0,0 +1,11 @@
+"""Hex colours program"""
+HEX_COLOURS = {"AliceBlue:""#f0f8ff", "Alizarin crimson:""#e32636", "Amethyst:""#9966cc", "Baby pink:""#f4c2c2",
```

Comment:
Dictionary entries should look like this: "AliceBlue":"#f0f8ff", not "AliceBlue:""#f0f8ff". I think you have created a set of strings instead of a dictionary.

Reply:
I think I miss typed when typing the dictionary. thank you for telling me though :)